source/help/getting-started/organizing-your-sidebar.rst

```diff
@@ -0,0 +1,176 @@
+
+To unmute the category, select the **Category options** icon in the sidebar, then select **Unmute Category**.
+
+    .. image:: ../../images/mute-categories.gif
+
+Sorting category channels
+-------------------------
+
+Using the Mattermost Web or Desktop App, you can control how channels are sorted for each category. Sort category channels manually (by dragging and dropping), alphabetically, or by recent conversations first.
+
+Select the **Category options** icon in the sidebar, then select **Sort** and choose from **Alphabetically**, **Recent Activity**, or **Manually**.
+
+    .. image:: ../../images/sort-categories.gif
+
+Grouping unread messages
+------------------------
+
+Catch up on all your unread channels in one place at the top of your sidebar with a one-click **Unreads** category.
+
+Go to **Main Menu > Account Settings > Sidebar**, set **Group unread channels separately** to **On**, then select **Save**.
+
+- When this setting is enabled, all unread messages appear only in the **Unreads** category.
+- When this setting is disabled, all unread messages appear within their respective categories and channels.
+
+When enabled, the **Unreads** category is organized as follows:
+
+- Unread messages that contain mentions are sorted to the top by most recent activity.
+- Unread messages that do not contain mentions are sorted by most recent and appear directly below unread messages containing mentions.
+- Muted channels that contain mentions are sorted by most recent and appear directly below unread messages without mentions.
+
+    .. image:: ../../images/unreads.gif
+
+.. tip::
+  If you prefer to see a decicated unread-only view in your sidebar, collapse all custom categories to show only unread messages, then disable **Grouping unread messages** under **Account Settings > Sidebar**.
+
+Sorting your Direct Messages
+----------------------------
+
+Sort your Direct Messages alphabetically or by recent conversations first.
+
+Select the **Channel options** icon in the sidebar, then select **Sort** and choose from **Alphabetically** or **Recent Activity**.
+
+How many Direct Messages to display?
+------------------------------------
+
+Control how many Direct Message conversations display in the **Direct Messages** category to keep your conversations manageable. You can choose to show all messages or a fixed number of messages.
+
+You have two ways to configure the number of Direct Messages to display:
+
+- Go to **Main Menu > Account Settings > Sidebar**, then set **Number of direct messages to show**.
+
+or
+
+- Select the **Channel options** icon in the sidebar, then select **Show**
+
+Choose from **All direct messages**, or choose to show **10**, **15**, **20**, or **40** messages.
+
+Once you exceed the number of Direct Messages configured, older messages are hidden from the **Direct Messages** category. You can increase the number of conversations displayed to see older Direct Messages.
```

Comment:
You can always increase the number of conversations displayed to see older Direct Messages.